### ISSUE TITLE
Simplify removing IPs from analyst firewall

### ISSUE THREAD

**Issue by defensivedepth** [Security-Onion-Solutions/securityonion#1344]
Need to make this a bit simpler with some sort of wrapper script.

**Removing hosts**
In order to remove a previously allowed host from accessing the manager you must first exclude the IP address from the group and allow Salt to remove it from the iptables. 
     sudo so-firewall --apply excludehost analyst 192.168.44.10

Next you will need to remove the IP address from the SLS file by typing the following command and pressing Enter:  
     sudo so-firewall removehost analyst 192.168.44.10

**Comment by weslambert** [comment]
Maybe a wrapper named`so-disallow`, which we have in the current platform?

**Comment by TOoSmOotH** [comment]
The issue here is not due to so-allow but due to the way we manage iptables rules in salt. If we just delete the host from there without disabling it, salt does not go and disable that rule right away.

**Comment by defensivedepth** [comment]
Understood, but we still need to wrap all this in a simple script

**Comment by jertel** [comment]
I suggested a similarly named high-level whitelist script called `so-deny` in https://github.com/Security-Onion-Solutions/securityonion/issues/4621.

**Comment by r3doubt** [comment]
I was trying to add an IP with so-allow and -a and hit a random character that was invalid along with the IP address.  I can't seem to remove that invalid address with above commands. (sudo so-firewall --apply excludehost analyst IPv4, sudo so-firewall removehost analyst IPv4 )  

I don't know the incorrect character that was entered along with the IP.  Tried the removal process using best guess of the incorrect character + IP, and just the IP with above commands.   Neither worked. Something in the code parsed out the invalid character when displaying it back to me, but I still saw error messages about an invalid character and "entry already present" when I try to add a new entry with the correct IP.  I'm working inside Proxmox VE
[

Some sort of script to make viewing and removing entries easier would be useful, or more documentation on which files and lines to edit.  I tried poking around /opt/so/ in various firewall yamls, but  I ended up deleting the manager node and starting over.  ]
![errorcode](https://user-images.githubusercontent.com/32580761/132775187-601d61fe-7345-4227-b61c-f7d9a1af6d29.PNG)
